index2.html

```diff
@@ -100,29 +100,28 @@ <h2>Our Services</h2>
       <h5 class="w3-center">Companies that trust us</h5>
       <!-- add a section here to have logos for companies we have worked with -->
       <div class="w3-row w3-padding-32">
-        <div class="w3-col l3 s6 w3-center">
-          <img src="images/naro.png" alt="Company 1" class="org-image" />
+        <div class="w3-col l2 s6 w3-center">
+          <img src="images/naro.png" alt="Company 1" class="partner-org-logo" />
         </div>
-        <div class="w3-col l3 s6 w3-center">
-          <img src="images/maaif.jpg" alt="Company 3" class="org-image" />
+        <div class="w3-col l2 s6 w3-center">
+          <img src="images/maaif.jpg" alt="Company 3" class="partner-org-logo" />
         </div>
-        <div class="w3-col l3 s6 w3-center">
-          <img src="images/pelum-logo.png" alt="Company 2" class="org-image" />
+        <div class="w3-col l2 s6 w3-center">
+          <img src="images/pelum-logo.png" alt="Company 2" class="partner-org-logo" />
         </div>
-        <div class="w3-col l3 s6 w3-center">
+        <div class="w3-col l2 s6 w3-center">
           <img
             src="images/hamwe.webp"
             alt="Company 4"
-            class="org-image"
-            style="margin-top: 13%"
+            class="partner-org-logo"
+            style="margin-top: 5%;"
```

Comment:
remove in-line styling - i mentioned this before. in-line styling is bad code and in the long run affects the efficiency of the system.

https://www.linkedin.com/advice/3/what-risks-using-inline-styles-html-skills-web-development-kip2c